meta-mbl-bsp-common/recipes-bsp/atf/atf.inc

```diff
@@ -50,7 +50,7 @@ ATF_COMPILE_FLAGS = "-C ${S} \
       BL32=${DEPLOY_DIR_IMAGE}/optee/tee-header_v2.bin \
       BL32_EXTRA1=${DEPLOY_DIR_IMAGE}/optee/tee-pager_v2.bin \
       BL32_EXTRA2=${DEPLOY_DIR_IMAGE}/optee/tee-pageable_v2.bin \
-      BL33=${DEPLOY_DIR_IMAGE}/u-boot.bin \
+      BL33=${DEPLOY_DIR_IMAGE}/${UBOOT_IMAGE} \
```

Comment:
Can you please add in the commit message an explanation about this change?

Reply:
OK. UBOOT_IMAGE is the right file in imx8mm case.